lib/list_element/list_element.dart

```diff
@@ -20,9 +37,23 @@ class _RandomWordsState extends State<RandomWords> {
                   style: _biggerFont,
```

Comment:
Single font styles may be left as-is in code, no need to put them into variables